docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -1,246 +1,211 @@
 .. _real_and_lazy_data:
 
+
+.. testsetup:: *
+
+    import dask.array as da
+    import iris
+    import numpy as np
+
+
 ==================
 Real and Lazy Data
 ==================
 
-What is Real and Lazy Data?
----------------------------
+We have seen in the :doc:`user_guide_introduction` section of the user guide that
+Iris cubes contain data and metadata about a phenomenon. The data attribute of a
+cube contains the actual numerical values recorded for the phenomenon the cube describes.
```

Comment:
You know what, that detail in this section is probably unnecessary. I'm going to remove it...